Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -364,13 +364,12 @@
 {
     CGPoint activePoint = currentPoint;
 
-    ScrollingTreeNode* root = scrollingTree()->rootNode();
-    if (!is<ScrollingTreeFrameScrollingNode>(root))
+    auto* rootNode = scrollingTree()->rootNode();
```

Comment:
We're no longer checking if `root` is a ScrollingTreeFrameScrollingNode here? why? I don't have expected a dynamicDowncast.

Reply:
Thanks for taking a look! As I noted in the commit message it already returns one. That was obscured previously due to it not using `auto*`.

Reply:
(See https://ews-build.webkit.org/#/builders/49/builds/23875/steps/13/logs/stdio for the compiler error I got, but you can also verify by looking at the type in `Source/WebCore/page/scrolling/ScrollingTree.h`.)

Reply:
You assume I read the commit log :P